AzureCalling/app/src/main/res/layout/fragment_teams_meeting.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Missing copyright tag. You can add to all the other xml files as well that are missing it.